index.html

```diff
@@ -45,7 +45,185 @@ <h1 class="banner_content">I’m Munish. <br>Glad to see you!</h1>
             </div>
         </section>
         
-        <!-- Header Banner Starts Here -->
+        <!-- Header Banner Ends Here -->
+
+        <!-- Works Section Starts Here -->
+
+        <section class="works">
+            <div class="work_card">
+                <div class="work_card_wrapper">
+                    <img src="images/work-1.png" alt="Work 1" class="work_img">
+                    <div class="work_card_content_wrapper">
+                        <h3 class="work_title">Tonic</h3>
```

Comment:
- [ ] Please updat the background color of the work section to match the figma design.

**Your project design**
![image](https://user-images.githubusercontent.com/74411135/235277139-0231e72a-952c-468b-9129-673786df3f0a.png)

**figma design**
![image](https://user-images.githubusercontent.com/74411135/235277164-9b1a6000-7ef1-48f2-a3fa-83c4bce0e7d2.png)

This will make your project way more prettier which will improve the user experience 📱  😄 👍